source/_components/media_player.toshiba.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Thosiba Cast TV"
+description: "Instructions on how to integrate a Toshiba Smart TV into Home Assistant."
+date: 2017-08-13 13:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+featured: false
+ha_release: 0.17
+ha_iot_class: "Local Polling"
+---
+
+The `toshiba` platform allows you to control a Toshiba Cast TV.
+
+Currently known supported models:
+
+- 65L621U
+
+If your model is not on the list then give it a test, if everything works correctly then add it to the list on [GitHub](https://github.com/home-assistant/home-assistant.io).
+
+This TV has a "standby mode" that need to be "active" to allow power-on/power-off
+
+This TV will also be listed as a Google Cast device with the discovery protocol.  This component allow to manage input, volume and power on/off.
```

Comment:
Does this component epxose any services?
If so you could document them in the same manor as https://home-assistant.io/components/apple_tv/#services